serviceuser.go

```diff
@@ -160,6 +161,23 @@ func (su *ServiceUser) waitUntilReady() error {
 	return nil
 }
 
+// Connect connects to the server at the given "host:port". Either Connect or
```

Comment:
Comment should start with ConnectWithTimeout.

Reply:
Fixed at f6f6d07704b9323da4735839ed0f1eee60b0f1f6